server/schedule/operator/builder.go

```diff
@@ -241,8 +262,9 @@ func (b *Builder) EnableForceTargetLeader() *Builder {
 	return b
 }
 
-func (b *Builder) setUseJointConsensus(useJointConsensus bool) *Builder {
```

Comment:
prefer to use two functions. e.g enable/disable

Reply:
It is only used in the test, so it will be complicated to change the test.

Reply:
I removed this function and adjusted the test code.